sql/updates/world/master/9999_99_99_00_world.sql

```diff
@@ -0,0 +1,6 @@
+
+ALTER TABLE `gameobject_template_addon` ADD COLUMN 
+    `AIAnimKitID` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `WorldEffectID`;
+
+ALTER TABLE `gameobject_addon` ADD COLUMN
+    `AIAnimKitID` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `WorldEffectID`;
```

Comment:
The common codestyle for single column changes or additions is a single query line. Multiline queries are used for multiple changes or multiple new entries.
```suggestion
-- 
ALTER TABLE `gameobject_template_addon` ADD COLUMN `AIAnimKitID` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `WorldEffectID`;
ALTER TABLE `gameobject_addon` ADD COLUMN `AIAnimKitID` INT(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `WorldEffectID`;
```
Also recommended: SQL comment line on top, in case the previous SQL file does not have a NewLine at the end. You can also choose to include descriptive comment text whenever needed.